biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2674,43 +2697,149 @@ subroutine terminate_patches(currentSite)
                       call fuse_2_patches(currentSite, youngerPatch, currentPatch)
                       gotfused = .true.
                    endif
+                endif               
+                ! The fusion process has updated the "younger" pointer on currentPatch
+                
+             endif ! older or younder patch
+          endif ! very small area
+
+        else !nocomp. We cannot fuse to patches with a different PFT identity in no competition mode. 
+          
+           ! Each patch has a PFT identity, and so cannot simply fuse to the older or younger patch
+           ! For each small current patch, we must first search older patch candidates, and then younger 
+           ! patch candidates. 
+           ! need to think about the youngest of PFT logic later. 
+
+           is_youngest = itrue !try and find a younger same-PFT patch
+           ! discover if this is the youngest patch of its PFT
+            fusingPatch => currentPatch%younger !if it's the youngest overall then it's defacto youngest of PFT
+            do while(associated(fusingPatch).and.is_youngest.eq.itrue)
+              if(fusingPatch%nocomp_pft_label.eq.currentPatch%nocomp_pft_label)then
+                is_youngest = ifalse ! we found a yonger patch, so this isn't the youngest one. 
+              endif ! PFT
+              if(associated(fusingpatch%younger))then
+                if(fusingpatch%patchno.eq.fusingpatch%younger%patchno)then
+                 write(*,*) 'is_youngest patch list error',fusingpatch%patchno,fusingpatch%younger%patchno
                 endif
-             endif
-          endif
-       endif
+               endif
+              fusingPatch => fusingPatch%younger 
+            enddo !fusing patch
+
+           is_oldest = itrue !try and find a younger same-PFT patch                                                 
+           !--------------------------------------------------
+           ! n.b. The following code is to figure out how to 
```

Comment:
this also seems like it should be subsumed under the anthro_dist_label logic?  the sum of all patch areas with a given nocomp_pft_label is fixed in time, so the only case where we should be able to run into logic with tiny patches that need to be force-merged is if there is a tiny patch with a potential fuser whose nocomp_pft_label matches but a anthro_dist_label that doesn't match? or am I thinking about this wrong?